tests/codeception/acceptance/LockupSettings/LockupSettingsCest.php

```diff
@@ -0,0 +1,368 @@
+
+  /**
+   * Test the lockup settings overrides.
+   */
+  public function testLockupSettingsS(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/config/system/lockup-settings');
+    $I->canSeeResponseCodeIs(200);
+    $I->uncheckOption('#edit-su-lockup-enabled-value');
+    $I->selectOption("#edit-su-lockup-options", "s");
+    $I->checkOption('#edit-su-use-theme-logo-value');
+    $I->fillField('Line 1', 'Site title line');
+    $I->fillField('Line 2', 'Secondary title line');
+    $I->fillField('Line 3', 'Tertiary title line');
+    $I->fillField('Line 4', 'Organization name');
+    $I->fillField('Line 5', 'Last line full width option');
+    $I->click('Save');
+    $I->amOnPage('/');
+    $I->canSee("Site title line");
+    $I->canSee("Secondary title line");
+    $I->canSee("Organization name");
+  }
+
+  /**
+   * Test the lockup settings overrides.
+   */
+  public function testLockupSettingsT(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/config/system/lockup-settings');
+    $I->canSeeResponseCodeIs(200);
+    $I->uncheckOption('#edit-su-lockup-enabled-value');
+    $I->selectOption("#edit-su-lockup-options", "t");
+    $I->checkOption('#edit-su-use-theme-logo-value');
+    $I->fillField('Line 1', 'Site title line');
+    $I->fillField('Line 2', 'Secondary title line');
+    $I->fillField('Line 3', 'Tertiary title line');
+    $I->fillField('Line 4', 'Organization name');
+    $I->fillField('Line 5', 'Last line full width option');
+    $I->click('Save');
+    $I->amOnPage('/');
+    $I->canSee("Site title line");
+    $I->canSee("Secondary title line");
+    $I->canSee("Tertiary title line");
+    $I->canSee("Organization name");
+  }
+
+  /**
+   * Test the logo image settings overrides.
+   */
+  public function testLogoWithLockup(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/config/system/lockup-settings');
+    $I->canSeeResponseCodeIs(200);
+    $I->uncheckOption('#edit-su-lockup-enabled-value');
+    $I->selectOption('#edit-su-lockup-options', 'a');
+    $I->fillField('Line 1', 'Site title line');
+    $I->fillField('Line 2', 'Secondary title line');
+    $I->fillField('Line 3', 'Tertiary title line');
+    $I->fillField('Line 4', 'Organization name');
+    $I->fillField('Line 5', 'Last line full width option');
+
+    // Add custom logo.
+    $I->uncheckOption('#edit-su-use-theme-logo-value');
+
+    // In case there was an image already.
+    try {
+      $I->click("Remove");
+    }
+    catch(Exception $e) {
+      // Do nothing and carry on.
+    }
+
+    try {
+      // For CircleCI
+      $I->attachFile('input[name="files[su_upload_logo_image_0]"]', '../acceptance/LockupSettings/logo.jpg');
+    }
+    catch(Exception $e) {
+      // For Local.
+      $uglyHack = "../../../../../../..";
+      $I->attachFile('input[name="files[su_upload_logo_image_0]"]', $uglyHack . __DIR__ . '/logo.jpg');
+    }
```

Comment:
Instead of all this, simply move the file into the correct place.

SeeL
https://github.com/SU-SWS/stanford_profile/pull/240/files/b199537f2b33039321d7c2b6f0146d9314091a29#diff-5b87d66a9862acf4dd752952797dc74f